src/Tasks/SynTree/Quiz.hs

```diff
@@ -28,7 +28,4 @@ generateSynTreeInst SynTreeConfig {..} = do
       { tree
```

Comment:
This whole module is also not needed anymore?

Reply:
Yes. I had to update some tests though.